Composer/cypress/integration/LUPage.spec.ts

```diff
@@ -15,12 +15,12 @@ context('LU Page', () => {
 
     cy.get('.toggleEditMode button').should('not.exist');
 
-    // by default is table view
+    // by default it goes to table view
     cy.findByTestId('LUPage').findByTestId('table-view').should('exist');
 
     // nav to ToDoBotWithLuisSample.main dialog
     cy.findByTestId('ProjectTree').within(() => {
-      cy.findByText('__TestToDoBotWithLuisSample').click();
+      cy.findByText('__TestToDoBotWithLuisSample').click(64, 0);
```

Comment:
Since this is not a typical usage of `click`, can you add a comment why this is necessary?

Reply:
Will do.

Reply:
Found another way to fix this.